ilc/server/i18n.js

```diff
@@ -4,8 +4,14 @@ const {intlSchema} = require('ilc-sdk/dist/server/IlcProtocol'); // "Private" im
 
 const i18nCookie = require('../common/i18nCookie');
 
+const isStaticFile = url => {
+    const extensions = ['.js', '.js.map'];
+
+    return extensions.some(ext => url.endsWith(ext));
+};
+
 const onRequestFactory = (i18nConfig, i18nParamsDetectionPlugin) => async (req, reply) => {
-    if (!i18nConfig.enabled || req.raw.url === '/ping') {
+    if (!i18nConfig.enabled || req.raw.url === '/ping' || isStaticFile(req.raw.url)) {
```

Comment:
Does req.raw.url includes query parameters? In case if yes the condition might not work.

Reply:
it includes, but the condition is just to reject `.js` files e.g. http://localhost:8233/_ilc/client.js

Reply:
if for some reason you'll get http://localhost:8233/_ilc/client.js?q=1412341 condition won't work.

Reply:
done https://github.com/namecheap/ilc/pull/407/commits/38331a0d3e3fe7577a06a3a6d81c68c8fbae007c